### PR TITLE
Move link text to translation file

### PR DESCRIPTION
**Pull request by nickbristow**

### CODE REVIEW

**Comment by jessieay** [comment]
@nickbristow CI is failing

I am totally down with small PRs, but might be helpful to do these in slightly larger chunks.

**Comment by jessieay** [comment]
This need some CI / Code Climate / Rebase ❤️  - LMK when it's ready for review!

File: app/assets/stylesheets/components/_lists.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.no-bullets {
+  list-style: none;
```
Comment (jessieay) [inline app/assets/stylesheets/components/_lists.scss]: we can use `.list-style-none` from bass cs for this!

File: app/views/home/_deploy.html.erb [hunk 2]
```diff
@@ -24,9 +24,12 @@
                 <%= link_to status.short_sha, status.commit_url %>
               </span>
             </div>
-            Deployed by <em><%= status.user %></em>
-            <%= time_ago_in_words(status.timestamp) %> ago.
-            View <%= link_to 'pending changes', status.pending_url %>.
+            <%= t('home.deployed_by_html',
+              user: status.user,
```
Comment (jessieay) [inline app/views/home/_deploy.html.erb]: possible to keep the `em` in the view? Ideally, we have as little HTML as possible in the translation files

File: app/views/service_providers/_form.html.slim [hunk 3]
```diff
@@ -8,14 +8,17 @@
 = form.input :description
 = form.input :issuer
 = form.association :agency
-= form.input :acs_url, label: 'Assertion Consumer Service URL'
-= form.input :assertion_consumer_logout_service_url, label: 'Assertion Consumer Logout Service URL'
-= form.input :sp_initiated_login_url, label: 'SP Initiated Login URL'
-= form.input :block_encryption, as: :radio_buttons, label: 'Block Encryption'
-= form.input :saml_client_cert, label: 'SAML Client Cert (PEM encoded)'
-= form.input :return_to_sp_url, label: 'Return to SP URL'
+= form.input :acs_url, label: t('service_providers.labels.acs_url')
```
Comment (jessieay) [inline app/views/service_providers/_form.html.slim]: these labels can be customized with simple_form

this is the default simple_form locale file: https://github.com/plataformatec/simple_form/blob/c3a307cb9501208e053f771738e34a3d93d33858/lib/generators/simple_form/templates/config/locales/simple_form.en.yml

if you copy it into `config/locales` you can add a label for this input like so (no need to set the `label` option, SimpleForm will find it for you!

```
# simple_form.en.yml

en:
  simple_form:
    labels:
      defaults:
         service_provider:
           acs_url: 'Assertion Consumer Service URL'
```

see how we did this for micropurchase here: https://github.com/18F/micropurchase/blob/f8b2f3d5c1b8c41a262e1ef2642764a395a88dc1/config/locales/defaults/simple_form.en.yml

File: app/views/service_providers/index.html.slim [hunk 4]
```diff
@@ -1,4 +1,4 @@
-h2 = t('headings.service_providers.mine')
+h2 = t('headings.service_providers.main')
```
Comment (jessieay) [inline app/views/service_providers/index.html.slim]: niiiiice 🍌

File: app/views/users/edit.html.slim [hunk 5]
```diff
@@ -1,4 +1,9 @@
-h2 = t('headings.users.edit', name: @user.email)
+h2 = t('headings.users.edit')
+
+ul.no-bullets
+  li = t('user.email_html', attr: @user.email)
```
Comment (jessieay) [inline app/views/users/edit.html.slim]: seems like this is adding content instead of I18n-ifying?

File: config/locales/en.yml [hunk 6]
```diff
@@ -34,20 +37,58 @@ en:
...
-      edit: 'Edit %{name}'
+      edit: 'Edit user'
     user_groups:
       index: 'User groups'
       sub_index: 'Manage user groups'
       edit: 'Edit user group'
       new: 'Create new user group'
 
+  user:
+    email_html: '<b>Email:</b> %{attr}'
```
Comment (jessieay) [inline config/locales/en.yml]: let's move this style stuff into the view, if we can!

File: app/assets/stylesheets/components/_lists.scss (hunk 1 above)
Comment (nickbristow) [inline app/assets/stylesheets/components/_lists.scss]: i missed that. yes

File: app/views/users/edit.html.slim (hunk 5 above)
Comment (nickbristow) [inline app/views/users/edit.html.slim]: yeah, i combined with another PR. 

I didn't like how the edit user page looked, I wanted to give the page more context.

Comment (jessieay) [inline app/views/users/edit.html.slim]: let's do it separately, please! PRs separated by functional changes are VERY helpful when looking through git history

File: app/views/home/_deploy.html.erb [hunk 7]
```diff
@@ -24,9 +24,12 @@
                 <%= link_to status.short_sha, status.commit_url %>
               </span>
             </div>
-            Deployed by <em><%= status.user %></em>
-            <%= time_ago_in_words(status.timestamp) %> ago.
-            View <%= link_to 'pending changes', status.pending_url %>.
+            <%= t('home.deployed_by_html',
+              user: "<em>#{status.user}</em>".html_safe,
```
Comment (zachmargolis) [inline app/views/home/_deploy.html.erb]: I think `content_tag(:em, status.user)` would be better -- doesn't need an explicit `.html_safe`

Comment (nickbristow) [inline app/views/home/_deploy.html.erb]: good point!

File: app/views/home/_deploy.html.erb [hunk 8]
```diff
@@ -24,9 +24,12 @@
                 <%= link_to status.short_sha, status.commit_url %>
               </span>
             </div>
-            Deployed by <em><%= status.user %></em>
-            <%= time_ago_in_words(status.timestamp) %> ago.
-            View <%= link_to 'pending changes', status.pending_url %>.
+            <%= t('home.deployed_by_html',
+              user: content_tag(:em, status.user),
```
Comment (jessieay) [inline app/views/home/_deploy.html.erb]: yaaaay

File: config/locales/en.yml [hunk 9]
```diff
@@ -1,57 +1,92 @@
...
+    service_providers_refresh_failed: Failed to publish service providers. Contact your administrator.
+    service_providers_refreshed: Success! Service providers publish request has been sent.
+  omniauth:
+    logout_fail: Logout failed
+    logout_ok: Successfully logged out
+  session_timedout: For your safety, we signed you out after being idle for %{session_timeout}. Please sign in again.
+  session_timeout_warning: We noticed you haven't been very active, hence we will sign you out in %{time_left_in_session}. Please click '%{continue_text}' to remain signed in.
+  shared:
+    official_site: An official website of the United States government
+  simple_form:
```
Comment (jessieay) [inline config/locales/en.yml]: I'd put this stuff in the `simple_form.en.yml` file

**Pull request merged**